catalog/dags/common/ingestion_server.py

```diff
@@ -4,17 +4,20 @@
 from urllib.parse import urlparse
 
 from airflow.exceptions import AirflowSkipException
+from airflow.models import Variable
 from airflow.providers.http.operators.http import SimpleHttpOperator
 from airflow.providers.http.sensors.http import HttpSensor
 from requests import Response
 
-from common.constants import XCOM_PULL_TEMPLATE
+from common.constants import ES_PROD_CONN_ID, XCOM_PULL_TEMPLATE
 
 
 logger = logging.getLogger(__name__)
 
 
 POKE_INTERVAL = int(os.getenv("DATA_REFRESH_POKE_INTERVAL", 60 * 15))
+# Minimum number of records we expect to get back from the API when querying an index.
+THRESHOLD_RESULT_COUNT = int(os.getenv("API_HEALTHCHECK_RECORD_COUNT", 10_000))
```

Comment:
This is no longer used in this PR, right? Can it be removed?

Reply:
Ah yes, thanks -- missed that one.